SmartCharger.Business/DTOs/RegisterDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Text.Json.Serialization;
+
+namespace SmartCharger.Business.DTOs
+{
+    public class RegisterDTO
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
```

Comment:
Could you explain me this part of the code and why do you use it?
[JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]

Reply:
The purpose of code
```c#
[JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
``` 
is to remove attribute which is null. For example if registration failed, response will not contain user like in this example:
```json
{
  "success": false,
  "message": "Registration failed.",
  "error": "Email is already in use.",
  "user": null
}
```
It will look like this
```json
{
  "success": false,
  "message": "Registration failed.",
  "error": "Email is already in use."
}
```

Reply:
That makes sense. Thank you.